src/drivers/px4io/CMakeLists.txt

```diff
@@ -60,9 +62,15 @@ add_custom_command(OUTPUT ${fw_io_exe}
 	COMMENT "Building ${config_io_board}"
 	USES_TERMINAL
 	)
+
+# only in the cygwin environment: convert paths for
+# binary in and output to mixed windows paths (C:/...)
+CYGPATH(fw_io_exe fw_io_exe_cyg)
+CYGPATH(fw_io_bin fw_io_bin_cyg)
+
 add_custom_command(OUTPUT ${fw_io_bin}
 	COMMAND mkdir -p ${PX4_BINARY_DIR}/ROMFS/${config_romfs_root}/extras/
-	COMMAND ${OBJCOPY} -O binary ${fw_io_exe} ${fw_io_bin}
+	COMMAND ${OBJCOPY} -O binary ${fw_io_exe_cyg} ${fw_io_bin_cyg}
```

Comment:
This is kind of nitpicky, but instead of having special cygwin handling all over the place can we sidestep the issue by using relative paths?

https://github.com/PX4/Firmware/blob/master/platforms/nuttx/NuttX/CMakeLists.txt#L56-L57

Reply:
good idea, I'll try